decidim-proposals/app/helpers/decidim/proposals/admin/proposal_bulk_actions_helper.rb

```diff
@@ -8,6 +8,30 @@ def proposal_find(id)
           Decidim::Proposals::Proposal.find(id)
         end
 
+        # Public: Generates a select field with the templates of the given component.
+        #
+        # component - A component instance.
+        # prompt - An i18n string to show as prompt
+        #
+        # Returns a String.
+        def bulk_templates_select(component, prompt, id: nil)
+          options_for_select = find_templates_for_select(component)
+          select(:template, :template_id, options_for_select, prompt:, id:)
+        end
+
+        def find_templates_for_select(component)
+          return @templates_for_select if @templates_for_select
```

Comment:
```suggestion
          return unless Decidim.module_installed? :templates
          return @templates_for_select if @templates_for_select
```
The module Templates might not be installed, so better check it too.